R/check_reliability.R

```diff
@@ -45,24 +43,24 @@ check_reliability.estimate_grouplevel <- function(x, ...) {
   if (length(unique(x$Level)) <= 3) {
     insight::format_alert(paste0(
       "The number of random levels (N = ",
-      length(unique(x$Level)),
+      insight::n_unique(x$Level),
```

Comment:
given that it makes the code longer so I'm not sure it's necessary here 😁

Reply:
it's more robust, as it deals with `NA` values :-)
and of course, our functions are always preferable.

Reply:
> and of course, our functions are always preferable.

Can't argue with that

![Uploading image.png…]()

Reply:
(and it has methods for lists and data frames, too)